src/tooltip/index.ts

```diff
@@ -36,6 +36,12 @@ export function getTooltip({ object }: GeoArrowPickingInfo): TooltipContent {
     return {
       className: "lonboard-tooltip",
       html: toHtmlTable(jsonObj),
+      style: {
+        backgroundColor: "#fff",
+        boxShadow: "0 0 15px rgba(0, 0, 0, 0.1)",
+        color: "#000",
+        padding: "10px",
```

Comment:
It looks a little cleaner with a smaller padding in my own opinion 🤷‍♂️ .
```suggestion
        padding: "6px",
```